fvm/errors/codes.go

```diff
@@ -75,6 +77,23 @@ const (
 
 	// contract errors 1250 - 1300
 	// ErrCodeContractError          ErrorCode = 1250 - reserved
-	ErrCodeContractNotFoundError      ErrorCode = 1251
+	ErrCodeContractNotFoundError ErrorCode = 1251
+
+	// errors no longer in use
+	// Deprecated: ErrCodeInvalidTxByteSizeError
+	ErrCodeInvalidTxByteSizeError ErrorCode = 1001
+	// Deprecated: ErrCodeInvalidReferenceBlockError
+	ErrCodeInvalidReferenceBlockError ErrorCode = 1002
+	// Deprecated: ErrCodeInvalidScriptError
+	ErrCodeInvalidScriptError ErrorCode = 1004
+	// Deprecated: ErrCodeInvalidGasLimitError
+	ErrCodeInvalidGasLimitError ErrorCode = 1005
+	// Deprecated: ErrCodeExpiredTransactionError
+	ErrCodeExpiredTransactionError ErrorCode = 1003
+	// Deprecated: ErrCodeEncodingUnsupportedValue
+	ErrCodeEncodingUnsupportedValue ErrorCode = 1102
+	// Deprecated: ErrCodeGasLimitExceededError
+	ErrCodeGasLimitExceededError ErrorCode = 1104
+	// Deprecated: ErrCodeContractNamesNotFoundError
```

Comment:
keeping them deprecated in there might help not to allocate the same number, this way you might reuse and not see it.

Reply:
thanks